web/shapeworks/src/views/Select.vue

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+import { defineComponent, onMounted } from '@vue/composition-api';
+import { getDatasets } from '@/api/rest'
+import { allDatasets, selectedDataset } from '../store/index';
+import { Dataset } from '@/types';
+import router from '@/router/routes';
+
+export default defineComponent({
+    setup() {
+        onMounted(async () => {
+            allDatasets.value = (await getDatasets()).sort((a, b) => {
+                if(a.created < b.created) return 1;
+                if(a.created > b.created) return -1;
+                return 0;
+            });
+        })
+        function selectDataset (dataset: Dataset) {
+            selectedDataset.value = dataset;
+            router.push('data')
+        }
+
+        return {
+            allDatasets,
+            selectedDataset,
+            selectDataset,
+        }
+    }
+})
+</script>
+
+<template>
+  <div class="flex-container pa-5">
+    <v-card
+        v-for="dataset in allDatasets"
+        :key="dataset.id"
+        class="dataset-card"
+    >
+        <div class="text-overline mb-4">
+            DATASET ({{ dataset.created.split('T')[0] }})
```

Comment:
Now that this isn't showing "DATASET <number>", the word `dataset` seems unnecessary.  And, since I generated a lot of my data with a loop, most of my data files have the same date.  Should we show the time (at least the hour and minute) as well as the date?  If so, can we have the browser use its local time zone?

This is petty and can be saved for a future PR.